README.md

```diff
@@ -1,25 +1,29 @@
 
-It enables the following usecases:
+With the `fermyon/actions` collection, you can incorporate [Spin](https://developer.fermyon.com/spin/index) and [Fermyon Cloud](https://developer.fermyon.com/cloud/index) in your [GitHub Action](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/configuring-a-workflow). 
 
-- setup spin and plugins (`fermyon/actions/spin/setup`)
-- build and push to OCI registry (`fermyon/actions/spin/push`)
-- deploy to Fermyon Cloud (`fermyon/actions/spin/deploy`)
-- deploy PR preview to Fermyon Cloud (`fermyon/actions/spin/preview`)
+This collection of Actions enables the following use cases:
 
-## `fermyon/action/spin/setup`
+- [x] set up Spin CLI and plugins using [`fermyon/actions/spin/setup@v1`](#install-spin-cli-and-plugins---fermyonactionsspinsetupv1)
+- [x] build and push your Spin app to an OCI registry using [`fermyon/actions/spin/push@v1`](#push-spin-app-to-an-oci-registry---fermyonactionsspinpushv1)
+- [x] deploy your Spin app to Fermyon Cloud using [`fermyon/actions/spin/deploy@v1`](#deploy-spin-app-to-fermyon-cloud---fermyonactionsspindeployv1)
+- [x] deploy PR preview to Fermyon Cloud [`fermyon/action/spin/preview@v1`](#deploy-preview-of-spin-app-to-fermyon-cloud---fermyon/action/spin/preview@v1)
+
+
+Let's take a look at each one to learn about the required inputs and walk through an example. 
+
+## Install Spin CLI and Plugins - `fermyon/actions/spin/setup@v1`
 
 setup `spin` with optional plugins
 
 ### Inputs
 
-| Name         | Required | Description                                                                                                                                   | Default |
-| ------------ | -------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
-| version      | Optional | The version of `spin` to install. Supports [semver](https://www.npmjs.com/package/semver) versioning                                          | latest  |
-| plugins      | Optional | The comma-separated list of `spin plugins` to install                                                                                         | -       |
-| github_token | Optional | The `GitHub` token for querying/downloading `spin` releases. If provided, it avoids Github api rate limiting during Github actions executions | -       |
+| Name         | Required | Default | Description                                                                                                                                   |
+| ------------ | -------- | ------- | --------------------------------------------------------------------------------------------------------------------------------------------- |
+| version      | False | latest       | The version of `spin` to install.                                         |
```

Comment:
I would recommend reverting the "required" column content to "Required/Optional" rather than "False/True."  This makes it more self-documenting for the reader, who doesn't need to scoot their eye up to the header to figure out _what_ is false or true, plus for cases like `undeploy` where the _value_ is a boolean it avoids potential mixups of "now am I looking at the default here."  This might seem silly for such small tables but I hit this speed-bump repeatedly while reading this!